src/contracts/pods/EigenPod.sol

```diff
@@ -667,20 +669,19 @@ contract EigenPod is IEigenPod, Initializable, ReentrancyGuardUpgradeable, Eigen
          * in the beacon chain as a full withdrawal.  Thus such a validator can prove another full withdrawal, and
          * withdraw that ETH via the queuedWithdrawal flow in the strategy manager.
          */
-            // if the withdrawal amount is greater than the MAX_VALIDATOR_BALANCE_GWEI (i.e. the max amount restaked on EigenLayer, per ETH validator)
-        uint64 maxRestakedBalanceGwei = _calculateRestakedBalanceGwei(MAX_VALIDATOR_BALANCE_GWEI);
```

Comment:
addressing the bug I found after the hexens report, where we send up to 1 ETH extra instantly to the podOwner vs sending it via the queuedWithdrawal system

Reply:
this check: https://github.com/Layr-Labs/eigenlayer-contracts/blob/9af7c3a4be085449ee6b4f54b89d5e0611314b85/src/test/EigenPod.t.sol#L1218